controllers/controllers/workloads/cfprocess_controller.go

```diff
@@ -103,6 +103,10 @@ func (r *CFProcessReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			}
 		}
 
+		var lrpHealthCheckPort int32 = 0
+		if len(cfProcess.Spec.Ports) > 0 {
+			lrpHealthCheckPort = cfProcess.Spec.Ports[0]
```

Comment:
Out of scope for this, but I still feel it's pretty strange to just pick the first element as the health check. We don't even know what type of traffic whatever is listening on that port expects.